Makefile

```diff
@@ -26,10 +26,19 @@ DRIVERWINDOWSBINARY=${DRIVERBINARY}.exe
 
 DOCKER=DOCKER_CLI_EXPERIMENTAL=enabled docker
 
+BASE_IMAGE_LTSC2019=mcr.microsoft.com/windows/servercore:ltsc2019
```

Comment:
It seems like you could split the base image and the tag, and then parameterize the build-and-push-windows rule, eg
build-and-push-windows-container-%: ...
  $(DOCKER) buildx .... -t $(STAGINGIMAGE):$(STAGINGVERSION)_$* \
      --build-arg BASE_IMAGE=$(WIN_BASE_IMAGE):$*

etc.

That might save copy-paste errors when updating things in the future. OTOH makefiles are complicated enough as it is, and being more explicit about what's going on isn't a bad idea.

Up to you.

Reply:
I looked into this but `--build-arg BASE_IMAGE=$(BASE_IMAGE_1909)` complicates this. Each base is going to be different and I want to have control over the specific version tag here so this is simpler.

Reply:
gotcha.

/lgtm

Reply:
/approved